experimental/plugins/postgres_storage/src/postgres_storage.rs

```diff
@@ -617,42 +633,81 @@ impl WalletStrategy for DatabasePerWalletStrategy {
     }
 }
 
+
+// determine additional query parameters based on wallet strategy
+fn get_multi_database_name(config: &PostgresConfig) -> &str {
+    // look to see if there is a specified db to use.  If not, use the default name
+    let name: &str = match config.database_name {
+        Some(ref database_name) => {
+            if database_name.is_empty() {
+                _WALLETS_DB
+            } else {
+                database_name
+            }
+        },
+        None => _WALLETS_DB,
+    };
+
+    return name;
+}
+
 impl WalletStrategy for MultiWalletSingleTableStrategy {
     // initialize storage based on wallet storage strategy
     fn init_storage(&self, config: &PostgresConfig, credentials: &PostgresCredentials) -> Result<(), WalletStorageError> {
+        match env_logger::try_init() {
+            Ok(_) => {}
+            Err(_) => {
+                debug!("failed to init logging, probably already initialized");
+            }
+        }
+        debug!("Entering init_storage");
         // create database and tables for storage
         // if admin user and password aren't provided then bail
         debug!("Initializing storage strategy MultiWalletSingleTableStrategy.");
         if credentials.admin_account == None || credentials.admin_password == None {
             return Ok(())
         }
 
+        debug!("setting up the admin_postgres_url");
+        // look to see if there is a specified db to use.  If not, use the default name
+        let wallet_db_name: &str = get_multi_database_name(config);
+        debug!("wallet_db_name: {:?}", wallet_db_name);
         let url_base = PostgresStorageType::_admin_postgres_url(&config, &credentials);
-        let url = PostgresStorageType::_postgres_url(_WALLETS_DB, &config, &credentials);
-
-        let conn = postgres::Connection::connect(&url_base[..], postgres::TlsMode::None)?;
+        let url = PostgresStorageType::_postgres_url(wallet_db_name, &config, &credentials);
+        debug!("postgres_url: {:?}", url);
+        debug!("connecting to postgres, url_base: {:?}", url_base);
+        let conn = postgres::Connection::connect(&url_base[..], config.tls())?;
```

Comment:
Why do you use here `config.tls()` as the last parameter but on the line 699 use `postgres::TlsMode::None` ? 
Is it intentional?  The same thing for `create_wallet`, `delete_wallet`

Reply:
changed them all to config.tls() to be consistent